general/press.rst

```diff
@@ -118,12 +118,36 @@ Blog Posts
   <http://www.cittametropolitanaroma.it/homepage/elenco-siti-tematici
   /wifimetropolitano/openwisp-la-soluzione-open-source-la-diffusione-
   servizi-wifi/>`_ (in Italian)
-- `Introducing OpenWISP Monitoring: Google Summer of Code 2020
-  Project report <https://medium.com/@nepython/openwisp-monitoring-
-  gsoc-2020-project-report-332441961629>`_
-- `GSOC 2017: adding AirOS support to netjsonconfig <https://edoput.github.io/openwispgsoc/>`_
-- `GSoC 2017 @ OpenWISP: Raspbian backend for OpenWISP 2
-  <https://medium.com/@ritwickdsouza/gsoc-openwisp-raspbian-backend-for-openwisp-2-61ff91843362>`_
+
+Google Summer of Code Blog Posts
+--------------------------------
+
+2017 Students
```

Comment:
we should put recent content first, the rest of the section should be organized this way as well, at least the presentations